src/components/Modeler.vue

```diff
@@ -327,6 +327,15 @@ export default {
     highlightNode(node) {
       store.commit('highlightNode', node);
     },
+    /**
+     * Register a mixin into a node component.
+     * Used during "modeler-before-init"
+     *
+     */
+    registerComponentMixin(component, mixin) {
```

Comment:
A suggestion: Since this method is fairly simple, do you think it makes more sense to add it as a parameter to `registerNode`? The one drawback of doing this is that many places may already be using `registerNode`, and modifying the function's signature will require a slight re-write of the places the function is called.

My reason for suggesting this is that, the only thing between `modeler-before-init` and `modeler-init` is a single function call, and the name isn't very meaningful, as it's still part of initialization.